### DISCUSSION TITLE
Does the bandwidth limit reset on a monthly basis?

### DISCUSSION THREAD

**Discussion by YO-SC** [vercel/community#5002] [Help] [answered]
### Summary

Does the consumed bandwidth reset to zero once the month concludes? The question goes for all tiers, I just want to be sure of it. Thanks.

### Steps to Reproduce

N/A

**Comment by amyegan** [comment, marked as answer]
Good question! Usage on the [Hobby plan](https://vercel.com/docs/accounts/plans/hobby) is calculated on a rolling 30 day basis. With this [30 day sliding window](https://vercel.com/docs/accounts/plans/hobby#hobby-billing-cycle) you drop one day of usage every day and there is not a complete usage reset event. It's good for personal projects and small-scale applications.

Usage on the [Pro plan](https://vercel.com/docs/accounts/plans/pro) is reported according to the monthly billing period instead. Meaning all of the past billing period usage is wiped away when the plan rolls over to a new month. It's good for professional developers, freelancers, and small businesses.

The [Enterprise plan](https://vercel.com/docs/accounts/plans/enterprise) is customizable so it can be tailored to the needs of large organizations. If you're interested in the Enterprise plan, it would be best to [contact our sales team](https://vercel.com/contact/sales) which can help you determine the right plan and pricing to suit your business's needs.

**Comment by YO-SC** [comment]
Alright, thanks @amyegan !
